Grigore Alexandru Cristian/3.8lab7/main.c

```diff
@@ -0,0 +1,58 @@
+#include <stdio.h>
+#include <stdlib.h>
+void readp(int grad1,int *p)
```

Comment:
This made me laugh 🤣 

How is it not the exact same function as `readq` but you'd just be passing different parameters?
This is the exact reason for which we have functions, such that we don't have to write a piece of code any time we want to only change the parameters!!